tests/b033.py

```diff
@@ -0,0 +1,17 @@
+"""
+Should emit:
+B033 - on lines 6-12
+"""
+
+test = {1, 2, 3, 3, 5}
+test = {"a", "b", "c", "c", "e"}
+test = {True, False, True}
+test = {None, True, None}
+test = {3, 3.0}
+test = {1, True}
+test = {0, False}
```

Comment:
Do True and False booleans hash to the same as 0 and !0 ... TIL if so. Guess I've never thought about it.

Reply:
Yup, I also discovered this when creating the check.

```python
test = {1, True}
print(test)
```
Will print:
```
{1}
```